metal/loadbalancers/emlb/emlb.go

```diff
@@ -21,10 +21,47 @@ func NewLB(k8sclient kubernetes.Interface, config string) *LB {
 }
 
 func (l *LB) AddService(ctx context.Context, svcNamespace, svcName, ip string, nodes []loadbalancers.Node) error {
+	tokenExchanger := &MetalTokenExchanger{
+		metalAPIKey: "TODO",
```

Comment:
Need to figure out how to pass this in.  I haven't looked yet at where the key comes from for CPEM and what changes would be needed to make it available here.